src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
```

Comment:
It's a good practice to group your CSS properties in a specific order, for example: positioning, box model, typography, visual, etc. Here, `display: flex;` should come before width and height properties.